src/librustc_back/target/windows_msvc_base.rs

```diff
@@ -53,6 +53,7 @@ pub fn opts() -> TargetOptions {
         exe_suffix: ".exe".to_string(),
         staticlib_prefix: "".to_string(),
         staticlib_suffix: ".lib".to_string(),
+        no_default_libraries: false,
```

Comment:
Oh. I completely missed that fact that this is a Windows target. This has nothing to do with rumprun, I will remove it. Sorry about that.

This slipped in from my previous [patch](https://github.com/rust-lang/rust/pull/28578). I ommited it in the final version, because the mvc linker doesn't include default libraries anyway.